src/Chapter3.hs

```diff
@@ -907,9 +1006,25 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = MkGold {unGold :: Integer} deriving (Show)
+
+instance Append Gold where
+  append :: Gold -> Gold -> Gold
+  append x y = MkGold (unGold x + unGold y)
+
+instance Append [a] where
+  append :: [a] -> [a] -> [a]
+  append x y = x ++ y
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
  append = (++)
```